tools/osbuild-mpp

```diff
@@ -1228,6 +1229,7 @@ class ManifestFile:
         name = desc.get("id", "image")
         self.vars[name] = Image.from_dict(desc)
 
+    # pylint: disable=no-self-use
```

Comment:
Would it be better to turn this into a `staticmethod` as well?

Reply:
Cannot change the prototype as other classes override it and they use `self`.